apps/api/src/app/benchmark/benchmark.service.ts

```diff
@@ -85,23 +85,23 @@ export class BenchmarkService {
 
       let performancePercentFromAllTimeHigh = 0;
 
-      if (allTimeHigh && marketPrice) {
+      if (allTimeHigh && allTimeHigh.marketPrice && marketPrice) {
         performancePercentFromAllTimeHigh = this.calculateChangeInPercentage(
-          allTimeHigh,
+          allTimeHigh.marketPrice,
           marketPrice
         );
       } else {
         storeInCache = false;
       }
-
       return {
         marketCondition: this.getMarketCondition(
           performancePercentFromAllTimeHigh
         ),
         name: benchmarkAssetProfiles[index].name,
         performances: {
           allTimeHigh: {
-            performancePercent: performancePercentFromAllTimeHigh
+            performancePercent: performancePercentFromAllTimeHigh,
+            date: allTimeHigh.date
```

Comment:
Can you move up to keep the alphabetical order?

Reply:
done